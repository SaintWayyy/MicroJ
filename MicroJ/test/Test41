int i : = 10;
int main(){
    for(i = 0; i < 10; i = i + 1){
        if(i == 4)
            break;
    }
    /*i should be 4 right now*/
    print("The answer of fib(5) is: ");
    print(fib(i));
}

int fib(int i){
    if(i == 1)
        return 1;
    if(i == 2)
        return 2;
    
    return fib(i - 1) + fib(i - 2);
}