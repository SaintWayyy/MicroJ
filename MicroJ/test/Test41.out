"The answer of fib(5) is: "
5
